### PR TITLE
Adds "A Stranger's Armor"

### PR DESCRIPTION
**Pull request by doshMobile**
spit this out five minutes before i had to run to whole foods im sorry for any dumbass code mistakes lskjflskdf

Adds the armor for the character "A Stranger," some goofy ass dude wandering around Lavaland.

aseprite_02_12_03 43 39PM

### CODE REVIEW

File: maplestation_modules/story_content/stranger_equipment/code/strangerclothing.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// funny lore man, overpowered so lavaland enemies/the crew dont kill him, won't attack crewmates so no worry about that. not meant to be worn by crewmembers
+/obj/item/clothing/suit/armor/strangerarmor
+	name = "stranger's armor"
+	desc = "A massive, bulky suit of armor. Resembles some sort of death knight."
```
Comment (Wallemations) [inline maplestation_modules/story_content/stranger_equipment/code/strangerclothing.dm:4]: Wow, Death Knight from the hit mobile game Raid Shadow Legends?!

Comment (doshMobile) [reply maplestation_modules/story_content/stranger_equipment/code/strangerclothing.dm:4]: i want you crucified

**Comment by doshMobile** [comment]
oh my god I can merge my own shit I forgot 
speedmerging because there's 20 minutes until the round and it'll be awkward if stranger joins butt ass naked

**Comment by CRITAWAKETS** [comment]
wow selfmerge smh

**Comment by doshMobile** [comment]
I am god

**Pull request merged**